network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/ThreeWindingsTransformerToInjectionAttributesAdapter.java

```diff
@@ -11,20 +11,15 @@
 import com.powsybl.network.store.model.LegAttributes;
 import com.powsybl.network.store.model.ThreeWindingsTransformerAttributes;
 
-import java.util.Objects;
-
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
 public class ThreeWindingsTransformerToInjectionAttributesAdapter extends AbstractIdentifiableToInjectionAttributesAdapter<ThreeWindingsTransformerAttributes> {
 
-    private final ThreeWindingsTransformerImpl.LegImpl leg;
-
     private final ThreeSides side;
 
-    public ThreeWindingsTransformerToInjectionAttributesAdapter(ThreeWindingsTransformerImpl.LegImpl leg, ThreeWindingsTransformerAttributes attributes, ThreeSides side) {
```

Comment:
Should we change that here ?

Reply:
It was to fix sonar issue : I changed the setBus method to remove the bus update notification (or else the tck tests in ConnectionNotificationTest.java fail because of the not expected extra notifications), and 'leg' field was not used in this class ...

Reply:
ok